sections/main-product.liquid

```diff
@@ -612,9 +622,9 @@
     "name": {{ product.title | json }},
     "url": {{ request.origin | append: product.url | json }},
     {% if seo_media -%}
-      {%- assign media_size = seo_media.preview_image.width | append: 'x' -%}
+      {%- assign media_size = seo_media.preview_image.width -%}
```

Comment:
same here, maybe remove the assign and use `seo_media.preview_image.width` directly if that works 👍

Reply:
I wasn't sure about these assign's within `<script type="application/ld+json">` on whether they needed to be like this. I'll test removing them, thanks.